packages/date-picker/src/vaadin-date-picker-mixin.js

```diff
@@ -84,6 +84,7 @@ export const DatePickerMixin = (subclass) =>
          */
         showWeekNumbers: {
           type: Boolean,
+          value: false,
```

Comment:
Added default value to avoid passing `undefined` to overlay content and month calendar elements.